### ISSUE TITLE
Research never unlocks

### ISSUE THREAD

**Issue by freaktechnik** [lainsce/reganam#22]
On 1.0.8 the synthesizer buttons never unlock for me, even though I have the research lab level and resources that are requested in the tooltip (level 1-3 and currently 800 crystal and hydrogen each, at a level 3 lab)

**Comment by freaktechnik** [comment]
After waiting longer to be much over target of the required resources the minerals synthesizer research button got enabled. Pretty sure the tooltip on the buttons is just wrong?

**Comment by lainsce** [comment]
This just got fixed in the master repo, I'm fixing the other issue in the list and it'll be released :+1:

**Comment by selwynorren** [comment]
Just built this from source on a different computer, and it all works really well, however it seems the minerals research does not get unlocked at all. I have my lab at level 2 800 Crystal and 800 Hydrogen and Crystal research has opened up, however Mineral research is still disabled.

What I did find interesting is if I used Dconf and reset the settings, then it works just fine. I will use teh planet buster and see if that work the value correctly
